imgui-render-dx10/manifest

```diff
@@ -0,0 +1,11 @@
+: 1
+name: imgui-render-dx10
+version: 1.86.0-a.0.z
+summary: imgui C++ library
```

Comment:
Same remarks as `imgui-core/manifest` , but also this needs a package-specific summary.